pkg/machines/components/memorySelectRow.jsx

```diff
@@ -51,7 +51,7 @@ class MemorySelectRow extends React.Component {
          * https://github.com/patternfly/patternfly-react/issues/3191
          */
         return (
-            <div className={'slider-input-group' + (readOnly ? ' disabled' : '')}
+            <div className='slider-input-group' disabled={readOnly}
```

Comment:
Same here

Reply:
yes, please revert. [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) is list of supported elements. Any other "disabled" elements is done through css.

Reply:
Can you please first fix places where you know it is not used correctly and then I'll review it once again. Thanks!